### ISSUE TITLE
All pools report 0% daily volume change

### ISSUE THREAD

**Issue by reuptaken** [Uniswap/uniswap-v3-info#91]
**Bug Description**

<img width="126" alt="Zrzut ekranu 2021-05-31 o 09 11 03" src="https://user-images.githubusercontent.com/26765602/120154696-6a05aa00-c1f0-11eb-8ea1-162d07cd8a48.png">

**Comment by willhennessy** [comment]
thanks reuptaken! It is fixed now. Please reopen if you see it again

**Comment by reuptaken** [comment]
Is it deployed? Because:

<img width="395" alt="Zrzut ekranu 2021-06-2 o 11 35 26" src="https://user-images.githubusercontent.com/26765602/120457427-b8ed4400-c396-11eb-82a9-48907e56027d.png">

**Comment by reuptaken** [comment]
Also: 
<img width="378" alt="Zrzut ekranu 2021-06-2 o 11 37 03" src="https://user-images.githubusercontent.com/26765602/120457627-e639f200-c396-11eb-81e9-18f04cf3faf4.png">

This is weird, hard to reproduce, but try to open one pool, then another pool and go back and forth using "back" / "forward" button in browser. Different values appear, changing after page is reloaded.

**Comment by willhennessy** [comment]
Good catch! The values change every time I refresh the page. It seems to always return one of four values, below. It's probably a reliability issue with the graph. Ian, can you take a look?

<img width="305" alt="Screen Shot 2021-06-03 at 11 54 10 PM" src="https://user-images.githubusercontent.com/5455577/120743645-6790a200-c4c7-11eb-92fc-ae31dd933313.png">
<img width="310" alt="Screen Shot 2021-06-03 at 11 54 22 PM" src="https://user-images.githubusercontent.com/5455577/120743646-6790a200-c4c7-11eb-8537-a8b6870ce95e.png">
<img width="307" alt="Screen Shot 2021-06-03 at 11 54 49 PM" src="https://user-images.githubusercontent.com/5455577/120743647-6790a200-c4c7-11eb-9ddc-69e84efaa9e4.png">
<img width="298" alt="Screen Shot 2021-06-03 at 11 55 14 PM" src="https://user-images.githubusercontent.com/5455577/120743648-68293880-c4c7-11eb-8628-e67b1ac29816.png">

**Comment by willhennessy** [comment]
I'm seeing this on the overview page too
<img width="1433" alt="Screen Shot 2021-06-04 at 12 01 04 AM" src="https://user-images.githubusercontent.com/5455577/120743969-00bfb880-c4c8-11eb-91c1-dea29890a083.png">

**Comment by ianlapham** [comment]
Should be the issue where time travel queries return same response ID - this was thought to be fixed in older PR but looking now we are still overwriting the query type in pool data - will fix to use 'no-cache'
